### DISCUSSION TITLE
Generators does not find artifacts from subdirs

### DISCUSSION THREAD

**Discussion by ivanvaccari** [loopbackio/loopback-next#7754] [Ideas]
### Suggestion
Generators may search for artifacts in the target folder *and** subdirectories

### Use Cases
Whan you have a lot of models(>70 in my case), you can group them in subfolders for better organization. But then generators cannot find them and, as example, relations cannot be genearted since the wanted models are not found.

### Other info
Im' aware of https://github.com/loopbackio/loopback-next/issues/6591 about the generation, i'm ok with this since i can always move the files after the generation, but then they become "lost" from a lb4 command point of view and cannot be used anymore for artifacts auto-generation.

**Comment by achrinza** [comment]
See: https://github.com/loopbackio/loopback-next/pull/7826
